### DISCUSSION TITLE
Windows Installer

### DISCUSSION THREAD

**Discussion by champs777** [nzbgetcom/nzbget#144] [Q&A] [answered]
I currently have the old NZBGet v21.1.  I am on a windows 10 machine.

I saw this article on how to backup and restore my settings.  I am good with this.
https://github.com/nzbgetcom/nzbget/discussions/100

Where can i find the v22 or higher of the new supported version?  I have looked and looked and I can't find it.

Thank you so much for your support!

**Comment by luckedea** [comment, marked as answer]
@champs777 All downloads are available under Releases page.
Direct link: https://github.com/nzbgetcom/nzbget/releases/tag/v22.0 under Assets tab (for version 22, it would get updates to v23 and further on)
For Windows you'd need this file: https://github.com/nzbgetcom/nzbget/releases/download/v22.0/nzbget-22.0-bin-windows-setup.exe

**Reply by champs777** [reply]
Thank you so much for the assistance!  I am sorry I totally looked right over that.
